src/logic/currentSession/store/reducer/currentSession.ts

```diff
@@ -22,5 +22,5 @@ export default handleActions(
       return newState
     },
   },
-  Map(),
+  makeCurrentSession(),
```

Comment:
can we go one step further and remove immutable.js there? this is one of our long-term goals https://github.com/gnosis/safe-react/issues/1090

Reply:
Tried with a more simple Redux approach. Removed immutable from current session